packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -549,4 +554,16 @@ class FlutterDriverExtension {
     }
     return SetSemanticsResult(semanticsWasEnabled != _semanticsIsEnabled);
   }
+
+  Future<void> _waitUntilCondition(bool condition(),
+      [Duration duration = const Duration(milliseconds: 10)]) {
```

Comment:
Why 10?

Reply:
Something that I feel like a reasonable interval to check the conditions... Not too frequent to keep this timer busy all the time; and still a reasonable gap before the state becomes stale. Any suggestions?

Reply:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-mysterious-and-magical-numbers-that-lack-a-clear-derivation

Reply:
Added comments on the default 10 milliseconds. PTAL.

Reply:
Regardless of the duration, polling seems wrong here - see my comment above about making this callback based.